CONTRIBUTING.md

```diff
@@ -20,16 +20,18 @@ Please contribute links to packages/projects you have used or are familiar with.
 
 ## Quality standard
 
-To stay on the list, package repositories should adhere to these quality standards:
+To be on the list, project repositories should adhere to these quality standards:
 
-- Generally useful to the community
-- Functional
-- Actively maintained (even if that just means acknowledging open issues when they arise)
-- Stable, or progressing toward stable
-- Documented (preferably godoc.org)
-- Tests are preferred (when possible)
+- Code is functional
+- Generally useful to the wider community of Go programmers
+- Actively maintained
+  - Regular, recent commits
+  - Or, for finished projects, issues and pull requests are responded to
+- Stable or progressing toward stable
+- Thoroughly documented (README, godoc comments, etc.)
+- Tests, where practical
```

Comment:
Suggestion. I think we should enforce it to follow [recommended Go style](https://github.com/golang/go/wiki/CodeReviewComments) (not necessarily perfectly, but to a large degree) and therefore have no or little `golint` issues.

The only exceptions should be existing packages that are extremely useful, popular and well known (but those tend to follow recommended Go style anyway).

I think high quality Go packages tend to already do this anyway, and this extra effort will filter out many unhelpful packages where authors did not put in a lot of time and effort or care into making the Go package as nice as it can be.